test/unit/enricher/page.spec.js

```diff
@@ -13,12 +13,26 @@ describe('PageEnricher', () => {
     useEach: true
   })
 
-  it('should return the url and the referrer of the page"', function () {
-    const state = {}
+  it('should return the url, referrer and the contextElements of the page', function () {
+    var newHeadline = document.createElement("h1")
+    var content = document.createTextNode("Some header")
+    newHeadline.appendChild(content)
+    var newP = document.createElement("p")
+    var newContentEmail = document.createTextNode("mailTo: john@test.com, also found: another@test.com !")
```

Comment:
AFAIK it is not valid to have whitespaces between 'mailTo:' and the email address. Also, usually `mailTo` is written as `mailto`.